test/integration/full/landmark-banner-is-top-level/frames/level1.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html id="pass2">
+<head>
+	<meta charset="utf8">
+	<script src="/axe.js"></script>
+</head>
+<body>
+	<p>This iframe should pass, too</p>
+
+	<div role="navigation">
```

Comment:
This markup is repeated more than once. Is there any way to reuse some of these integration test HTML files? Maybe with conditional JS?